### PR TITLE
Fix typo arguement --> argument

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

## Was the change discussed in an issue?
fixes #???

### CODE REVIEW

File: pkg/occlient/occlient.go [hunk 1]
```diff
@@ -2840,7 +2840,7 @@ func tar(tw *taro.Writer, fileName string, destFile string) error {
 
 // recursiveTar function is copied from https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/cp.go#L319
 func recursiveTar(srcBase, srcFile, destBase, destFile string, tw *taro.Writer) error {
-	glog.V(4).Infof("recursiveTar arguements: srcBase: %s, srcFile: %s, destBase: %s, destFile: %s", srcBase, srcFile, destBase, destFile)
+	glog.V(4).Infof("recursiveTar arguments: srcBase: %s, srcFile: %s, destBase: %s, destFile: %s", srcBase, srcFile, destBase, destFile)
```
Comment (kadel) [inline pkg/occlient/occlient.go]: 👍

File: pkg/occlient/occlient.go [hunk 2]
```diff
@@ -2775,7 +2775,7 @@ func makeTar(srcPath, destPath string, writer io.Writer, files []string) error {
 	srcPath = path.Clean(srcPath)
 	destPath = path.Clean(destPath)
 
-	glog.V(4).Infof("makeTar arguements: srcPath: %s, destPath: %s, files: %+v", srcPath, destPath, files)
+	glog.V(4).Infof("makeTar arguments: srcPath: %s, destPath: %s, files: %+v", srcPath, destPath, files)
```
Comment (kadel) [inline pkg/occlient/occlient.go]: 👍

File: vendor/github.com/kubernetes-incubator/service-catalog/cmd/svcat/broker/deregister_cmd.go [hunk 3]
```diff
@@ -46,7 +46,7 @@ func NewDeregisterCmd(cxt *command.Context) *cobra.Command {
 	return cmd
 }
 
-// Validate checks that the required arguements have been provided
+// Validate checks that the required arguments have been provided
```
Comment (kadel) [inline vendor/github.com/kubernetes-incubator/service-catalog/cmd/svcat/broker/deregister_cmd.go]: We should make changes in vendored source code. This needs to be fixed upstream.

Comment (mik-dass) [reply vendor/github.com/kubernetes-incubator/service-catalog/cmd/svcat/broker/deregister_cmd.go]: @kadel We **shouldn't** right?

Comment (kadel) [reply vendor/github.com/kubernetes-incubator/service-catalog/cmd/svcat/broker/deregister_cmd.go]: > @kadel We **shouldn't** right?

ah, yes, we should NOT 😊

**Comment by apupier** [comment]
> @apupier Can you please revert changes in files in vendor directory?
> 
> We shouldn't make any changes to this, this is vendored core from other repositories and it needs to be updated there, not in our copy.

done

sorry I forgot about this unbelievable convention of duplicating the world

**Comment by kadel** [comment]
Thank you @apupier

**Pull request merged**